### PR TITLE
SLVS-1359 Improve usability experience

### PR DESCRIPTION
**Pull request by gabriela-trutan-sonarsource**

### CODE REVIEW

**Comment by georgii-borovinskikh-sonarsource** [comment]
I don't think forcing all windows to open at the application center is good. We can center only the main 2 windows to the application, and then center the following windows according to the Binding/ConnectionManagement window locations

File: src/ConnectedMode.UnitTests/UI/OrganizationSelection/OrganizationSelectionViewModelTests.cs [hunk 1]
```diff
@@ -124,4 +124,20 @@ public void IsValidSelectedOrganization_IgnoresName(string key, string name, boo
 
         testSubject.IsValidSelectedOrganization.Should().Be(expectedResult);
     }
+
+    [TestMethod]
+    public void NoProjectExists_NoConnections_ReturnsTrue()
```
Comment (vnaskos-sonar) [inline src/ConnectedMode.UnitTests/UI/OrganizationSelection/OrganizationSelectionViewModelTests.cs:129]: I sense some copy-pasting here 😂 

`NoConnections` --> `NoOrganizations`

File: src/ConnectedMode.UnitTests/UI/OrganizationSelection/OrganizationSelectionViewModelTests.cs [hunk 2]
```diff
@@ -124,4 +124,20 @@ public void IsValidSelectedOrganization_IgnoresName(string key, string name, boo
...
+    {
+        var viewModel = new OrganizationSelectionViewModel([]);
+
+        viewModel.NoOrganizationExists.Should().BeTrue();
+    }
+
+    [TestMethod]
+    public void NoProjectExists_HasConnections_ReturnsFalse()
```
Comment (vnaskos-sonar) [inline src/ConnectedMode.UnitTests/UI/OrganizationSelection/OrganizationSelectionViewModelTests.cs:137]: `HasConnections` -->`HasOrganizations`

**Review by vnaskos-sonar — APPROVED** [review]
👍

**Pull request merged**